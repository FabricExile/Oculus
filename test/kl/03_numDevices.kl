require Oculus;

operator entry() {
  ORDevice device();
  report(device.numDevices());
}