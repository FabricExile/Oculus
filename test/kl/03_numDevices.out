[FABRIC:MT] Oculus: Oculus Rift initialized.
[FABRIC:MT] 1
[FABRIC:MT] Oculus: Oculus Rift finalized.
